src/App.tsx

```diff
@@ -1,6 +1,15 @@
 import { useEffect } from "react";
 import { Routes, Route } from "react-router-dom";
-import { Navbar, LandingPage, SingleCategoryPage } from "./Components";
+import {
+  Navbar,
+  LandingPage,
+  SingleCategoryPage,
+  Profile,
+  PrivateRoute,
+  Login,
+  SignUp,
+  QuizContainer,
+} from "./Components";
 import { API_URL } from "./utilities/constants";
 import { useQuizData } from "./Context/QuizContext/QuizDataProvider";
```

Comment:
If you want to avoid writing long lines for imports, you can use a combination of absolute imports and barrels.